### ISSUE TITLE
Problem with mongo ruby driver integration

### ISSUE THREAD

**Issue by benedikt** [appsignal/appsignal#92]
I discovered an issue with the mongo ruby driver integration. Both in production and development (when the agent is active), I get this error:

```
$ rails c
Loading development environment (Rails 4.2.5)
[1] pry(main)> Site.first # Trigger some query
NoMethodError: undefined method `[]=' for nil:NilClass
from /Users/benedikt/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/bundler/gems/appsignal-7ef57b2e322c/lib/appsignal/integrations/mongo_ruby_driver.rb:12:in `started'
[2] pry(main)> Appsignal::VERSION
=> "1.0.2.beta.2"
[3] pry(main)> 
```

Unfortunately, this breaks the whole application as it can't connect to the database. 
This happens with both `1.0.1` from RubyGems and the latest version from `master`.

**Comment by matsimitsu** [comment]
Thanks for reporting @benedikt I'll keep you posted on the progress.

**Comment by matsimitsu** [comment]
hi @benedikt we've fixed this issue and will release this on `1.0.2` this week.
